internal/commands/app/diff_test.go

```diff
@@ -367,4 +374,14 @@ func TestAppDiffInputs(t *testing.T) {
 			tc.test(t, tc.inputs, profile)
 		})
 	}
+
+	t.Run("should return an error when specified local path does not exist", func(t *testing.T) {
+		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
+		defer teardown()
+
+		localPath := "fakePath"
+
+		i := diffInputs{LocalPath: localPath}
+		assert.Equal(t, errInvalidPath(localPath), i.Resolve(profile, nil))
```

Comment:
if we do go with the extra returned error I suggest above, let's make sure to add a similar test case for that scenario